lib/graphql/resolver.ex

```diff
@@ -932,6 +932,10 @@ defmodule AshGraphql.Graphql.Resolver do
     child_complexity + 1
   end
 
+  def fetch_dataloader(loader, api, batch_key, parent) do
```

Comment:
this should be `defp`

Reply:
I've tried to put `to_resolution(Dataloader.get(...))` into the callback function, but when middleware cannot call it because `to_resolution` is private